www/templates/diary/trip_list_item.html

```diff
@@ -0,0 +1,105 @@
+<div>
+<!-- New Trip Template format  -->
+
+    <ion-item id="unified-diary-item" style="background-color: transparent;" class="list-item">
+
+        <!-- Start Time Tag -->
+        <div class="start-time-tag">{{trip.display_start_time}}</div>
+
+        <!-- div id="diary-card" ng-class="trip.listCardClass" ng-style="{'height': '{{itemHt}}px', 'margin-bottom': '1px'}"> -->
+        <div class="diary-card">
+            <!-- Leaflet Map -->
+            <leaflet
+            class="diary-map"
+            geojson="tripgj"
+            defaults="mapCtrl.defaults"
+            id="$index"
+            data-tap-disabled="true"
+            height="342px">
+            
+            </leaflet>
+
+            <!-- Trip Information -->
+            <div class="diary-infos" style="padding-top: 10px">
+
+                <div>test</div>
+                <div>test</div>
+                <div> {{trip.isDraft}}</div>
+                
```

Comment:
Why do we have this placeholder code here in a finished PR?

Reply:
Removed now.

Reply:
Confirmed this as removed

_**Confirmed this change as complete**_